dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/JdbiRepositoryImpl.java

```diff
@@ -454,4 +456,23 @@ protected Integer retrieveNextSortIndexForParentChildren(
     }
     return sortIndex;
   }
+
+  protected UUID[] extractUuids(List<? extends UniqueObject> uniqueObjects) {
```

Comment:
```suggestion
  protected UUID[] extractUuids(Collection<? extends UniqueObject> uniqueObjects) {
```
Ich weiß nicht mehr, ob wir das schon probiert hatten, allerdings sind sowohl `List` als auch `Set` von `Collection` abgeleitet. Da müsste dann diese eine Methode ausreichend sein.

Reply:
ist umgebaut